src/assets/i18n/en.json

```diff
@@ -5,7 +5,8 @@
     "MENU": {
         "HOME": "Home",
         "ABOUT": "About",
-        "VISIT": "Visit"
+        "VISIT": "Visit",
+        "HISTORY": "History"
```

Comment:
maybe this should be called 'Patient's history' or 'Previous appointments' ....

Reply:
or VISIT_HISTORY